docs/USAGE-GUIDE.md

```diff
@@ -251,6 +251,29 @@ typedef enum { S2N_STATUS_REQUEST_NONE, S2N_STATUS_REQUEST_OCSP } s2n_status_req
 status request an S2N_CLIENT should make during the handshake. The only
 supported status request type is OCSP, **S2N_STATUS_REQUEST_OCSP**.
 
+
+```c
+typedef enum { S2N_CERT_AUTH_NONE, S2N_CERT_AUTH_REQUIRED } s2n_cert_auth_type;
+```
+**s2n_cert_auth_type** is used to declare what type of client certificiate authentication to use.
+Currently the default for s2n is for neither the server side or the client side to use Client (aka Mutual) authentication.
+
```

Comment:
`s2n_cert_auth_type` and `s2n_mode` are orthogonal, and any combination of {`S2N_CERT_AUTH_NONE`, `S2N_CERT_AUTH_REQUIRED`} x {`S2N_SERVER`, `S2N_CLIENT`} can be used to alter which transitions in the TLS State machine are taken. The default for both Server and Client mode is `s2n_cert_auth_type` = `CERT_AUTH_NONE`, so neither side will use Client Auth by default.